client/app/scripts/superdesk/editor/editor.js

```diff
@@ -487,10 +489,14 @@ angular.module('superdesk.editor', [])
                 };
 
                 scope.addWordToDictionary = function() {
-                    var editorElem = elem.find(scope.type === 'preformatted' ?  '.editor-type-text' : '.editor-type-html');
                     var word = scope.replaceTarget.textContent;
                     spellcheck.addWordToUserDictionary(word);
-                    editorElem.trigger('input');
+
+                    for (var i = 0; i < editor.elements.length; i++){
+                        var editorElem = editor.elements[i].find(
```

Comment:
Perhaps consider using `forEach()` here for looping through the Array, more concise:
```js
editor.elements.forEach(function (editorEelem) {
    ...
});
```